pkgs/build-support/setup-hooks/wrap-gapps-hook/tests/lib.nix

```diff
@@ -19,12 +18,14 @@ rec {
   '';
 
   expectSomeLineContainingYInFileXToMentionZ = file: filter: expected: ''
-    if ! cat "${file}" | grep "${filter}"; then
-        ${fail "The file “${file}” should include a line containing “${filter}”."}
+    file=${lib.escapeShellArg file} filter=${lib.escapeShellArg filter} expected=${lib.escapeShellArg expected}
```

Comment:
bash syntax here seems odd to me.

Reply:
Why?

Reply:
I just didn't know that multiple variables could be assigned like that in a row. I'd separate them for readability.

Reply:
I think it's just obvious plumbing code that should take as little space as possible, so I'll wait for a third vote.

Reply:
In fact this should be such a common pattern that I'm considering implementing something like `${lib.mkShellVars [ "file" "filter" "expected" ]}`

Reply:
> I think it's just obvious plumbing code that should take as little space as possible, so I'll wait for a third vote.

It's OK, keep it that way.